static/js/user_profile.js

```diff
@@ -184,9 +184,13 @@ export function show_user_profile(user) {
     $("#user-profile-modal-holder").html(render_user_profile_modal(args));
     overlays.open_modal("user-profile-modal", {autoremove: true});
     $(".tabcontent").hide();
-    $("#profile-tab").show(); // Show general profile details by default.
+    if (default_tab === 1) {
+        $("#user-profile-streams-tab").show();
+    } else {
+        $("#profile-tab").show(); // Show general profile details by default.
+    }
```

Comment:
Can we replace this block with just calling the function to select the target tab after or when initializing the tab widget? It seems a bit weird to do be duplicating a bit of the `callback` logic defined for the tab switcher with the couple lines above this.

Reply:
Done.